pkgs/by-name/go/go-enum/package.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
```

nit: `finalAttrs` allows for overriding with [`buildGoModule`](https://github.com/NixOS/nixpkgs/issues/315337#issuecomment-2138814370)
```suggestion
buildGoModule (finalAttrs: {
```